authorization/permissions.py

```diff
@@ -19,27 +27,32 @@ class FilterBackend(object):
     """
     FilterBackend interface
     """
-    def filter_queryset(self, request, queryset, view):
+    def filter_queryset(
+            self,
+            request: HttpRequest,
+            queryset: QuerySet[Any],
+            view: Any,
```

Comment:
There was previous discussion about the `Any` type in view parameters. How is it going to be?